Introduction To Wireless Networking (#1377)
section-engineering-education/engineering-education: articles/introduction-to-wireless-networking/index.md

### Introduction

> Wireless networks have exploded the world currently, all the same, we all hate wires all over, or don't we? Or perhaps not!  

Reviewer: Change the introductory sentence. Make it a bit more formal.

It has even replaced desktops that had connectors (RJ-45) on the wall.  
Okay, let's say you go to a big hotel or airport, they will have a wireless network that you can use to access the internet.  
The world has undergone a lot of internet evolution. 
I would tell you this for sure if you wanted to buy a laptop that had the specification of a desktop with an RJ-45 connector on the wall,
you would pay the double price.  

> Nowadays, the laptop price is affordable, or isn't it?  

Reviewer: This sentence does not make sense.

> 

Reviewer: Pay attention to the formatting.

![image](https://user-images.githubusercontent.com/26842636/107795096-43754d80-6d7e-11eb-83e2-ddcfe6ba2d95.png)

It looks odd if each sentence occupies a line each.

Reviewer: Make the change throughout the article.

In this article, we are going to discuss the following:

* Definition of wireless networking
* How wireless network works.
* Types of wireless networks
* Wireless networking standards  
* Benefits of wireless networking

### A Quick look at networking

Computer network refers to devices, such as computers, fax or even printers connected or linked to each other to communicate and share resources. It makes use of transmission media such as wires and cables.
For example, in Local Area Networks (LAN), computers are interconnected in a small area such as an office via communication cables.  

The main reason for connecting several devices on a network is to share resources. For example, an office has one printer shared, it can be connected to all computers within this office on a network to allow usage by each department.  

> You ought to have also learned that computer networks have five components.  

Reviewer: ```suggestion
Computer networks have five components.
- message
- sender
- receiver
- protocol
- transmission medium
```

> They include  ***message, sender, receiver, protocol, and Transmission media***.  

Reviewer: ```suggestion

```

Reviewer: do you mean transmission medium?

### What's wireless networking?

Wireless networking refers to the technology where devices communicate without being tethered to any wires or cables.  
As we have already learned, in wired networks, computers are connected physically via transmission media,  
which is not the case with Wireless networks.  
Now, wired networks have one drawback, that is, your desktop or laptop has to be physically connected or tethered   
to a router or access switch, let's say via ethernet, minimizing the portability of your computers.  

> Obviously, in wireless, we don't use cables but radio waves instead.

Reviewer: ```suggestion
In wireless networks, we use radio waves instead of cables.
```

> But do you remember **Carrier-sense multiple access with collision detection (CSMA/CD)** used in half-duplex networks?    

Reviewer: Assume the reader has no knowledge of CSMA and CSMA/CD. You need to explain these.

Wireless networking is always half-duplex where the transmission can only take place in a single direction.   

> The main reason why wireless is half-duplex is because of sending and receiving on a single frequency.    

Reviewer: ```suggestion
The reason why wireless is half-duplex is that sending and receiving happens on the same frequency. 
```

Discussing this always leads to arguments on collisions, but that's beyond the scope of this article.  

> ### How does a wireless network works?

Reviewer: How does a wireless network work?

> While learning about computer networks, you might have come across radio frequency technology.    

Reviewer: Don't come across as expecting the reader to know everything in the article. Using phrases such as you might have come across, it is obvious, etc., sounds discouraging. Instead, use encouraging phrases. I have observed this throughout the article.

Well, if you haven't, then let's remind ourselves of what it means and its role in wireless networking.

***Radio Frequency wireless technology***

> Radio Frequency (RF) refers to the electromagnetic signals sent wirelessly to allow for communication.  

Reviewer: Use a proper definition here for RF.

> These signals are unique and, therefore, cannot interfere with each other.

Reviewer: What do you mean the signals are unique?

For more information on radio frequencies, click [here](https://eu.mouser.com/applications/rf-wireless-technology/#:~:text=A%20radio%20frequency%20(RF)%20signal,from%203kHz%20to%20300%20GHz.)

Now that we've got a little knowledge of Radio Frequencies, a point to note 
is that wireless networks depend on this technology to accomplish their tasks. 

> >This technology is beyond the scope of this article. Therefore we won't discuss it in-depth, but you're free to find good books on the topic. 

Reviewer: Why is this under a quote section?

Reviewer: Suggest resources.

For wireless networks to work, a device is known as `Access Point(AP)` is required.   

The ***Access Point (AP)*** has its sole responsibility to broadcast(distribute) the signals so that the devices can detect and connect to them.  
Have you ever wondered how public ***Wi-Fi***, e.g., the ones in hotels, schools, or airports, work? 

> How it works is that wireless networks create a two-way data communication channel.    

Reviewer: Don't begin a sentence with how, why, what, etc.,
